bin/init.sh

```diff
@@ -35,7 +35,7 @@ GO_TOP=$(cd $(dirname $0)/../../../..; pwd)
 
 export OUT_DIR=${OUT_DIR:-${GO_TOP}/out}
 
-HELM_VER=v2.7.2
+HELM_VER=v2.9.1
```

Comment:
I saw we require helm 2.9.1 now, should we update our doc in istio.io for it?

Reply:
Well - I am just trying to see if 2.9.1 works - with CRDs set as pre-init hook. 
2.7.2 is a bit old - I don't know what helm users do, but the best practice is to keep current.s